workers/unity/Assets/Playground/Scripts/Worker/MobileClientWorkerConnector.cs

```diff
@@ -38,5 +65,20 @@ public override void Dispose()
 
             base.Dispose();
         }
+
+        public Option<string> GetHostIp()
+        {
+            return ipAddress;
+        }
+
+        public Option<string> GetDevAuthToken()
+        {
+            return Option<string>.Empty;
```

Comment:
this should try to read the dev auth token from a txt asset

Reply:
I thought I added and pushed this 🤔

Reply:
Ah I had it in a different place.